tasks/users-docker.yml

```diff
@@ -0,0 +1,14 @@
+# file: postgresql/tasks/users-docker.yml
```

Comment:
to support creating users(+privilegies) below, we could also extend the original includes, doing the changes I proposed in existing open PR: https://github.com/ANXS/postgresql/pull/166/files

As long as it's only 1 duplicated task here (and below for privileges) , I'm ok also with the solution here